lib/.eslintrc.yaml

```diff
@@ -80,7 +82,9 @@ rules:
     - name: Uint8ClampedArray
     - name: URIError
     - name: WeakMap
+      into: Safe
     - name: WeakSet
+      into: Safe
```

Comment:
You need to consider the case below. We can't use `into` option to force prefix.
`into` option is just a prefix for reporting. It doesn't affect static analysis logic.

```js
      {
        code: `
          const { Map } = primordials;
          new Map({})
        `,
        options: [{ name: 'Map', into: 'Safe' }],
        errors: [{ message: /const { SafeMap } = primordials/ }]
      },
```

Reply:
I'm not sure we want to do that – I guess there are times we want to use `Map` instead of `SafeMap`.

Reply:
Addressed in https://github.com/nodejs/node/pull/36026, I've added a test that covers this.